src/resolve/adapters/digitalExperienceSourceAdapter.ts

```diff
@@ -147,7 +148,9 @@ export class DigitalExperienceSourceAdapter extends BundleSourceAdapter {
     // 3 because we want 'digitalExperiences' directory, 'baseType' directory and 'bundleName' directory
     const basePath = pathParts.slice(0, typeFolderIndex + 3).join(sep);
     const bundleFileName = pathParts[typeFolderIndex + 2];
-    const suffix = this.isBundleType() ? this.type.suffix : this.registry.getParentType(this.type.id)?.suffix;
+    const suffix = ensureString(
```

Comment:
this will throw, so ex: if the metadataRegistry ever loses a suffix for DEB stuff, it'll fail tests